doc/programming_guide/media.rst

```diff
@@ -95,25 +119,102 @@ DirectSound is available only on Windows, and is installed by default.
 pyglet uses only DirectX 7 features. On Windows Vista, DirectSound does not
 support hardware audio mixing or surround sound.
 
+.. _guide-openal:
+
 OpenAL
 ^^^^^^
 
-OpenAL is included with Mac OS X. Windows users can download a generic driver
-from `openal.org`_, or from their sound device's manufacturer. Most Linux
-distributions will have OpenAL available in the repositories for download.
-For example, Arch users can ``pacman -S openal`` and Ubuntu users can ``apt install libopenal1``.
+By default, this is the most preferred driver. It will be used first
+if it is installed unless overridden. See :ref:`guide-audio-driver-order`
+to learn more.
 
-Pulse
-^^^^^
+This driver is currently the best choice for POSIX systems because:
+
+* It's either preinstalled or easy to install.
+* It implements features which may be absent from OS-specific APIs,
+  such as positional audio
+
+OpenAL's main downside is that platforms other than Mac OS X are not
+guaranteed to have it preinstalled.
+
+Windows users can download a generic driver from either `openal.org`_
+or their sound device's manufacturer.
+
+On Linux, it may already be installed. It is strongly recommended to
+install it since the :ref:`guide-pulseaudio` backend has serious bugs
+and limitations. The commands to install OpenAL on the most common
+Linux distros are listed below. You may need to use ``sudo`` to run
+them.
+
+.. list-table::
+    :header-rows: 1
+
+    * - Common Linux Distros
+      - Install Command
+
+    * - Ubuntu, Pop!_OS, Debian
+      - ``apt install libopenal1``
+
+    * - Arch, Manjaro
+      - ``pacman -S openal``
+
+    * - Fedora, Nobara
+      - ``dnf install openal-soft``
+
+These commands should be safe to run when OpenAL is already
```

Comment:
I would say that it's not necessary to explain this part, since it's dependent on the package manager (and they'll all pretty much say the same thing anyway).
I also wouldn't bother with the RHEL comment.